MarvelApiClient/CharactersApiClient.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CharactersApiClient.swift
+//  MarvelApiClient
+//
+//  Created by Pedro Vicente on 14/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BrightFutures
+
+public class CharactersApiClient : BaseApiClient {
+
+    private let parser: CharactersParser
+
+    init(baseEndpoint: String, timeProvider: TimeProvider, httpClient: HttpClient, parser: CharactersParser) {
+        self.parser = parser
+        super.init(baseEndpoint: baseEndpoint, timeProvider: timeProvider, httpClient: httpClient)
+    }
+    
+    public func getAll(offset: Int, limit: Int) -> Future<GetCharactersDTO,NSError> {
+        let params = [MarvelApiParams.Offset : "\(offset)", MarvelApiParams.Limit : "\(limit)"]
```

Comment:
Does negative value of offset and int are authorized ?
Should we filter them in the client or let them fail in server ?